Example/NWSTokenView/Views/NWSImageToken/NWSImageToken.swift

```diff
@@ -1,49 +0,0 @@
-//
-//  NWSImageToken.swift
-//  NWSTokenView
-//
-//  Created by James Hickman on 8/11/15.
-/*
-Copyright (c) 2015 NitWit Studios, LLC
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.//
-*/
-
-import UIKit
-import NWSTokenView
-
-public class NWSImageToken: NWSToken
```

Comment:
@wfilleman I would prefer to leave this in as a proper example for subclassing the NWSToken as most users will want to customize their own tokens and not just used my default one (that was the primary reason I created this control). Having this in here gives a proper example of what is needed to subclass NWSToken and make it more customizable.